docs/sources/live/live-feature-overview.md

```diff
@@ -15,7 +15,7 @@ As soon as there is a change to the dashboard layout, it is automatically reflec
 
 ## Data streaming from plugins
 
-With Grafana Live data source, plugins can stream data updates in the form of Grafana data frames to a frontend.
+With a help of Grafana Live data source plugins can stream data updates in the form of Grafana data frames to a frontend.
```

Comment:
```suggestion
With Grafana Live, backend data source plugins can stream updates to frontend panels.
```

maybe?

Reply:
@FZambia, does this suggestion clarify things?

Reply:
Yes, previously reader could think that there is some sort of `Grafana Live data source` which is confusing.